src/pattern-library/components/Library.js

```diff
@@ -96,7 +96,7 @@ function PageIntro({ children }) {
  */
 function Section({ children, id, intro, title }) {
   return (
-    <section className="pb-16 space-y-8">
+    <section className="my-16 space-y-8">
```

Comment:
I agree, but nearly got sucked into a vortex/rathole on Friday afternoon trying to address this broadly. This change you cited actually _decreases_ some of the spacing because it allows for margin-collapse.

The broader "fix" requires focus and re-jiggering of the whole cascade of components and spacing and margins. It may have to wait until the next jog unless I can find some extra time.

Reply:
https://github.com/hypothesis/frontend-shared/issues/600